conda/meta.yaml

```diff
@@ -7,7 +7,8 @@ source:
     path: ../
 
 build:
-    script: python -m pip install --no-deps --ignore-installed .
+    noarch: python
+    script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
```

Comment:
This is for lines 26 to 31: You need to add `sshtunnel` as a requirement for the tests.